packages/backend/src/server/web/feed.ts

```diff
@@ -2,20 +2,22 @@ import { Feed } from 'feed';
 import { In, IsNull } from 'typeorm';
 import config from '@/config/index.js';
 import { User } from '@/models/entities/user.js';
-import { Notes, DriveFiles, UserProfiles, Users } from '@/models/index.js';
+import { Users, Notes, DriveFiles, UserProfiles } from '@/models/index.js';
```

Comment:
Not sure why you reordered these imports.

Reply:
somethine was added after v12 lol